app/todo_lists/resources.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, TodoList
+from app.lib.response_util import buildOkResponse
+
+
+def _returnTodoList(todoList):
+    """Private method to convert a todoList object into a dictionary.
+    """
+    return {
+        "id": todoList.id,
+        "name": todoList.name
+    }
+
+
+class TodoListMultiResource(Resource):
+    """Class for creating and accessing lists.
+    """
+    def get(self):
+        """Method gets a list of todoLists and returns them in an Ok response.
+        """
+        todoLists = TodoList.query.all()
+
+        return buildOkResponse([_returnTodoList(todoList) for todoList in
+                               todoLists])
+
+    def post(self):
+        """Method creates a todoList and returns it in an Ok response.
+        """
+        newListData = json.loads(request.form['data'])
```

Comment:
Please also add a little validation to the form data given and raise any exceptions you think are necessary.